filters/glitch/src/GlitchFilter.js

```diff
@@ -0,0 +1,269 @@
+import {vertex} from '@tools/fragments';
+import fragment from './glitch.frag';
+
+/**
+ * The GlitchFilter applies a glitch effect to an object.<br>
+ * ![original](../tools/screenshots/dist/original.png)![filter](../tools/screenshots/dist/glitch.png)
+ *
+ * @class
+ * @extends PIXI.Filter
+ * @memberof PIXI.filters
+ * @param {object} options - The more optional parameters of the filter.
+ * @param {number} [options.slices=5] - The count of slices.
+ * @param {number} [options.offset=100] - The max-offset of slices.
+ * @param {number} [options.direction=0] - The angle in degree of the offset of slices.
+ * @param {number} [options.fillMode=0] - The fill mode of the space after the offset.
+ *                 0: TRANSPARENT; 1: ORIGINAL; 2: LOOP; 3: CLAMP; 4: MIRROR.
```

Comment:
Could you add `GlitchFilter.TRANSPARENT`, `GlitchFilter.ORIGINAL` etc as static constants that can be referenced by developers? Easier to use than remember an integer.

Reply:
How to define static property ? 
I use 
```
export default class GlitchFilter extends PIXI.Filter {

    static TRANSPARENT = 0;
    static ORIGINAL = 1;
    static LOOP = 2;
    static CLAMP = 3;
    static MIRROR = 4;
```
but Failed

Reply:
You can define them outside of the class. Or as `static get` methods.

Reply:
have to use ```
    GlitchFilter.TRANSPARENT = 0;
    GlitchFilter.ORIGINAL = 1;
    GlitchFilter.LOOP = 2;
    GlitchFilter.CLAMP = 3;
    GlitchFilter.MIRROR = 4;
```
So ugly...

Reply:
DONE.